app/Main.hs

```diff
@@ -1,5 +1,25 @@
 module Main where
-import qualified PureScm.CoreFn as CoreFn
+
+import           Data.Text.IO                                  as T (putStrLn)
+import qualified Data.Aeson                                    as JSON
+import qualified Data.Aeson.Types                              as JSON.T
+import qualified Language.PureScript.CoreFn.FromJSON           as CoreFn
+import           Language.PureScript.CoreFn.Module             (Module)
+import           Language.PureScript.CoreFn.Ann                (Ann)
+import           Language.PureScript.Scheme.CodeGen.Transpiler (moduleToScheme)
+import           Language.PureScript.Scheme.CodeGen.Printer    (printScheme)
 
 main :: IO ()
-main = CoreFn.readJSONFile "resources/corefn.json"
+main = scheme >>= T.putStrLn
+
+readJSONFile :: FilePath -> IO (Module Ann)
+readJSONFile path = do
+  r <- JSON.decodeFileStrict' path
+  let res = r >>= JSON.T.parseMaybe CoreFn.moduleFromJSON
+  case res of
+    Nothing -> error "Nope"
+    Just (version, module_) -> return module_
+
+readJSON = readJSONFile "resources/PureScmTest.Literals.corefn.json"
```

Comment:
It's good practice to have signatures for all top-level identifiers (for documentation and sanity reasons, also the compiler will complain)